public/js/index.js

```diff
@@ -30,6 +30,7 @@ const lessonInput = document.querySelector(".create-lesson-input");
 const formElement = document.querySelector("form");
 const submitLessonElement = document.querySelector("#submit");
 const lessonsContainer = document.querySelector(".lessons");
+const LessonCount = document.querySelector(".lessons-count");
```

Comment:
In JS it's convention to only capitalise Classes and constructor functions. Could you change this to `lessonCount` please.

Reply:
oh yeah didn't catch that, I will